src/styles/index.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+export const color = Object.freeze({
+  casper: '#B0B9D3',
```

Comment:
I'd prefer to keep colors named according to how they're used in the design system

Reply:
Yeah, I'm open to suggestions here.

Reply:
@markmalazarte Do you have any insight on the, uh, semantics of the colors in our current designs?  Do you have a way you think about each of these colors, and decide when they are appropriate, or something like this?

Reply:
I don't have a hard stance on naming conventions but recently I have been creating a custom Semantic UI theme and using their naming conventions for their color variables and that seems to be good for me. You can still have something like `@casper: #BB0B9D3;` but then do something like `@lightTextColor: @capser` or something along these lines.

Reply:
I'll have a look at that.  That makes sense!

Reply:
I've posted the snippet in the `#front_end_arch` slack channel along with suggestions
...I'd rather handle all the variables in either one place I'm not quite sure how `styles/index.js` works